seed/static/seed/js/controllers/column_mappings_controller.js

```diff
@@ -317,4 +319,29 @@ angular.module('BE.seed.controller.column_mappings', [])
           _.values(grouped_by_from_field), function (group) {return group.length > 1})
         );
       };
+
+      $scope.preset_action_ok = (action) => {
+        if ($scope.current_preset.preset_type === seedConstants.PRESET_TYPE_NORMAL) {
+          return true
+        }
+
+        if ($scope.current_preset.preset_type === seedConstants.PRESET_TYPE_BUILDINGSYNC_DEFAULT) {
+          return false
+        }
+
+        if ($scope.current_preset.preset_type === seedConstants.PRESET_TYPE_BUILDINGSYNC_CUSTOM) {
+          const allowed_actions = [
+            'update',
+            'rename',
+            'delete',
+            'change_to_field',
+            'change_from_units',
+            'remove_column',
```

Comment:
I might be misremembering our conversation, but I thought we wanted to disallow the removal of columns for non-default BS presets?

![Screen Shot 2020-05-13 at 2 21 33 PM](https://user-images.githubusercontent.com/30608004/81861502-912a0480-9525-11ea-8170-21aadcb77eff.png)

Reply:
Right, I will remove this option